### DISCUSSION TITLE
Copy Paste Feature

### DISCUSSION THREAD

**Discussion by hagerp** [statamic/cms#4757] [Q&A]
Hi we are considering to use Statamic for our new main CMS for our agency.

We are missing the following feature / or just dont find it.

Is it possible top copy paste an entry for example from the page builder (peak starter kit)? 

![copypaste](https://user-images.githubusercontent.com/35568411/142173103-b2993595-cf3f-49da-99fb-aa87f1dfd5a0.jpeg)

thanks
Phil

**Comment by jackmcdade** [comment]
At this time there isn't a native way to copy and paste a Replicator Set, but perhaps you could copy and paste the content inside it?
